test/basic.html

```diff
@@ -55,22 +60,24 @@
           assert.equal(collapse.horizontal, false);
         });
 
-        test('default opened height', function(done) {
-          setTimeout(function() {
-            // store height
-            collapseHeight = getComputedStyle(collapse).height;
-            // verify height not 0px
-            assert.notEqual(collapseHeight, '0px');
-            done();
-          }, delay);
+        test('default opened height', function() {
+          // verify height not 0px
+          assert.notEqual(collapseHeight, 0);
+        });
+
+        test('opened height gets updated when inner dom changes', function() {
+          var lorem = collapse.children[1];
+          collapse.removeChild(lorem);
+          assert.isBelow(collapse.offsetHeight, collapseHeight);
+          collapse.appendChild(lorem);
+          assert.equal(collapse.offsetHeight, collapseHeight);
         });
 
         test('set opened to false', function(done) {
           collapse.opened = false;
           setTimeout(function() {
-            var h = getComputedStyle(collapse).height;
             // verify height is 0px
-            assert.equal(h, '0px');
+            assert.equal(collapse.offsetHeight, 0);
```

Comment:
It's probably better to use getBoundingClientRect, since it's already officially supported in different browsers https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect